crates/control/src/motion/motor_command_collector.rs

```diff
@@ -1,32 +1,19 @@
-use color_eyre::{eyre::WrapErr, Result};
+use color_eyre::Result;
```

Comment:
I call it `motor_commands`, because it contains the joint angles *and* the stiffnesses, I therefore think this better describes the content as `joint_command`. But it also doesn't make sense to rename the type, therefore the difference here.

Reply:
Renamed JointsCommand into MotorCommand